packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -488,15 +488,15 @@ function validate_shadow_output(result) {
 	if (headers instanceof Headers) {
 		if (headers.has('set-cookie')) {
 			throw new Error(
-				'Shadow endpoint request handler cannot use Headers interface with Set-Cookie headers'
+				'Endpoint request handler cannot use Headers interface with Set-Cookie headers'
 			);
 		}
 	} else {
 		headers = lowercase_keys(/** @type {Record<string, string>} */ (headers));
 	}
 
 	if (!is_pojo(body)) {
-		throw new Error('Body returned from shadow endpoint request handler must be a plain object');
+		throw new Error('Body returned from endpoint request handler must be a plain object');
```

Comment:
So... this is a bit clunky but strictly speaking I think we need something like this, since standalone endpoint request handlers can return either a POJO or any valid `Request` body. Thoughts?

```suggestion
		throw new Error('Body returned from page's endpoint request handler must be a plain object');
```

Reply:
Yeah, I guess that's a fair point, we _do_ already separate out 'standalone endpoints' and document the extra flexibility they have. So it's probably fine as-is. Thanks for the gut check!